Branch-SDK-TestBed/src/main/java/io/branch/branchandroiddemo/MainActivity.java

```diff
@@ -109,6 +110,7 @@ public void onInitFinished(JSONObject referringParams, BranchError error) {
                         if (error != null) {
                             Log.e("BranchSDK_Tester", "branch set Identity failed. Caused by -" + error.getMessage());
                         }
+                        Toast.makeText(getApplicationContext(), "Set Identity to test_user_10", Toast.LENGTH_SHORT).show();
```

Comment:
It would be good to add an input for the id instead of hardcoded. (as well as showing what the current id is)